Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md

```diff
@@ -0,0 +1,469 @@
+  
+To make the desired color into a lighter form we can use “Opacity” as a property for the elements.
+The opacity property specifies the opacity/transparency of an element. It can take a
+value from 0.0 - 1.0. The lower value, the more transparent:  
+  
+
+![](images/Screenshot%202021-09-29%20091241.jpg " Picture") 
+  
+  (PC. from w3school.com) [Reference link](https://www.w3schools.com/css/css_background.asp)
+  
+We use RGBA model instead of RGB functional notation model and it is also included a fourth component: the alpha channel (or opacity).
+The alpha channel is a number between 0.0 (fully transparent) and 1.0 (fully opaque). We can also specify a percentage where 0% is the same as 0.0 and 100% is the same as 1.0.
+For example, a bright red that's 50% opaque can be represented as rgb(255, 0, 0, 0.5)or rgb(100%, 0, 0, 50%).
+
+**In HEX string notation model, we use it as "#rrggbbaa" OR "#rgba" for opacity property.** 
+Specifies a color whose red component is the hexadecimal number 0xrr, green component is 0xgg, and blue component is 0xbb. The alpha channel is specified by0xaa; the lower this value is, the more translucent the color becomes.
+
+As an example, you can represent the opaque color bright blue as "#0000ff" or "#00f". To make it 25% opaque, one can use "#0000ff44" or "#00f4". 
+  
+We can also define “opacity” property for an element in CSS like as following:
+
+Eg.
+
+```  
+.
+.
+.
+ div {
+
+background-color: green;
+opacity: 0.6;
+  
+ }
+ 
+ .
+ .
+ . 
+```
+ Output:
+  
+ 
+![](images/Screenshot%202021-09-29%200912411.jpg " Picture")
+ 
+  
+  
+## 3. The ways of adding colors in CSS
+•	**External CSS:** Specifying colors in stylesheets. External styles are defined within the `<link>` element, inside the `<head>` section of an HTML file.
+
+•	**Internal CSS:** Adding all the required properties via `<style>` tag within `<head>` section of an HTML file.
+
+•	**Inline CSS:** An inline style may be used to apply a unique style for a single element.  
+  
+# 4.Things that can have color in CSS
+
+At a fundamental level, the color property defines the foreground color of an HTML element's content and the ***background-color*** property defines the element's background color. Thesecan be used on just about any element.
+
+## **i) Text**
+
+Whenever an element is rendered, these properties are used to determine the color of the text, its background, and any decorations on the text.
+
+**Color:** The color to use when drawing the text and any text decorations (such as the addition of under- or overlines, strike-through lines, and so forth.
+
+Eg. 
+
+
+
+```
+<h1 style="color: Tomato;">Hello World</h1>
+```
+Output:  
+  
+
+![](images/Screenshot%202021-09-28%20132016.jpg " Picture")
+  
+**background-color:** The text's background color.
+
+
+
+```
+<h1 style="background-color:DodgerBlue;">Hello World</h1>
+```
+Output:
+  
+![](images/Screenshot%202021-09-28%20132141.jpg " Picture")  
```

Comment:
While adding images, please name correctly.
Add alt text as well. Title can be optional.